price-list-overview.md update (#584)
bigcommerce/dev-docs: docs/api-docs/catalog/price-list-overview.md

# Price List API
<div class="otp" id="no-index">
	<h3> On this page </h3>
	<ul>
        <li><a href="#what-is-a-price-list?">What is a Price List?</a></li>
        <li><a href="#price-list-definitions">Price List definitions</a></li>

>         <li><a href="#price-list-assignments">Price List Assignments</a></li>

Reviewer: ```suggestion
        <li><a href="#price-list-assignments">Price List assignments</a></li>

```

# Price List API
<div class="otp" id="no-index">
	<h3> On this page </h3>
	<ul>
        <li><a href="#what-is-a-price-list?">What is a Price List?</a></li>
        <li><a href="#price-list-definitions">Price List definitions</a></li>

>         <li><a href="#price-list-assignments">Price List assignments</a></li>

Reviewer: ```suggestion
        <li><a href="#price-list-assignments">Price List Assignments</a></li>

```

        <li><a href="#price-list-notes">Price List notes</a></li>
        <li><a href="#price-list-resources">Price List resources</a></li>
	</ul>
</div>

## What is a Price List?

A Price List allows you to populate different versions of catalog pricing and assign them to different [Customer Groups](/api-reference/customer-subscribers/customers-api). The prices are specified exclusively at the variant level.

> If an active Price List does not contain prices for a variant, then the Catalog pricing will be used. You can associate a Price List to a Customer Group via the Control Panel or the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)

Reviewer 2: Hi @traci707 and @ektravel!  Another way to make the association here is through price list assignments. 
 https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-assignments/createpricelistassignments

> Price Lists will provide overridden price values to the Stencil storefront. You can further customize the final price display within the Stencil template. For more information, see the [Price Object](https://developer.bigcommerce.com/stencil-docs/reference-docs/global-objects-and-properties) in the Stencil documentation.

Reviewer: The link does not take you to the Price Object directly. My suggestion is to rename the link to Theme Objects. 
```suggestion
Price Lists will provide overridden price values to the Stencil storefront. You can further customize the final price display within the Stencil template. For more information, see the [Theme Objects](https://developer.bigcommerce.com/stencil-docs/reference-docs/global-objects-and-properties).

```

### OAuth scopes
The following OAuth Scopes are required:
* [Products](/api-docs/getting-started/basics/authentication#authentication_oauth-scopes)

## Price List definitions

> - `Price List` -- A collection of price records. `Price Records` make up a price list.

Reviewer: Removed code font as it does not seem to be necessary in this case.
```suggestion
- `Price List` -- A collection of price records. Price records make up a price list.

```

```

> - `PriceRecordBatch` -- A way to update several `Price Records` in a `Price List` at once. Using this bulk upsert endpoint, you can upsert up to 1000 Price Records in a single API call.

Reviewer: Removed code font as it does not seem to be necessary in this case.
```suggestion
- `PriceRecordBatch` -- A way to update several price records in a `Price List` at once. Using this bulk upsert endpoint, you can upsert up to 1000 price records in a single API call.

```

Reviewer: Alternatively, both mentions can be done in code font.

> - `Currency` --  A `Price List` can contain records for multiple currencies. *Only price records that match the store's default currency determine storefront and in-cart prices.* Although BigCommerce supports a storefront currency selection, this is not currently integrated with Price Lists and will merely convert prices from the store's default currency for display convenience.

Reviewer 2: We will use price records for transactional currencies.  If the merchant's default us USD and they have EUR set up as a transactional currency in the control panel, when a shopper selects EUR we will look for EUR price records.  If those don't exist, we fall back to catalog pricing and convert it.  

If the alternate currencies are display only, we'll read the store default currency price records and convert for display.

Under `discount_rules` the `type` is set to `price_list` and the `price_list_id` is 1. Which is the id of the price list the group has been assigned to.

> ## Price List Assignments

Reviewer: ```suggestion
## Price List assignments

```

> Price List Assignment assigns a Price List to a specific sales Channel. This association lets you define custom pricing for shoppers only on a specific external site or platform. Price List Assignments combined with a Customer Group assignment allows you to target specifically more logged in customers shopping on that Channel.

Reviewer: Added code font for consistency with the rest of the document.
```suggestion
Price List Assignment assigns a `Price List` to a specific sales channel. This association lets you define custom pricing for shoppers only on a specific external site or platform. Price List Assignment, combined with a Customer Group, allows you to target specifically more logged in customers shopping on that Channel.

```

Reviewer: I think the word channel can be all lowercase in this instance. 
```suggestion
Price List Assignment assigns a Price List to a specific sales Channel. This association lets you define custom pricing for shoppers only on a specific external site or platform. Price List Assignments combined with a Customer Group assignment allows you to target specifically more logged in customers shopping on that channel.

```

```

## Price List notes

- Bulk Pricing Tiers may additionally be associated with a Price Record to indicate different pricing as the quantity in cart increases.

- If a variant has a `Price Record`, any existing product-level bulk pricing will not apply in the cart.  For variants without `Price Records`, any existing product bulk pricing will apply.

> - Price Lists Records accepts bulk upsert. You can only do one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) at a time. Running more than one in parallel on the **same store** will cause a 429 error, and the request will fail.

Reviewer: Added code font.
```suggestion
- `Price Lists Records` accepts bulk upsert. You can only do one [Bulk upsert](https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-records/setpricelistrecordcollection) at a time. Running more than one in parallel on the **same store** will cause a 429 error, and the request will fail.

```

## Price List resources

### Related endpoints
* [Get Price List Collection](/api-reference/catalog/pricelists-api/price-lists/getpricelistcollection)

### Webhooks available

> There are no direct webhooks available for Price Lists. Since Price Lists directly relate to products, webhooks related to products will fire for corresponding changes such as pricing.

Reviewer 2: A price record update will not trigger product or variant webhooks.  Webhooks for price lists and price records are on our roadmap, but no idea when that will be done.
